dist/apache.conf

```diff
@@ -0,0 +1,142 @@
+	</Directory>
+
+	# Additional configuration
+	<Directory /home/mastodon/live/public>
+		Require all granted
+	</Directory>
+
+	RewriteEngine On
+	# Serve local files if present, but otherwise proxy them
+	RewriteCond %{DOCUMENT_ROOT}/%{REQUEST_FILENAME} !-f
+	RewriteCond %{REQUEST_URI} !/acme-challenge/
+	RewriteCond %{REQUEST_URI} !^/api/v1/streaming
+	RewriteRule ^/(.*)$ http://127.0.0.1:3000%{REQUEST_URI} [P,L]
+	# Add some more needed headers
+	RequestHeader set X-Forwarded-Proto expr=%{REQUEST_SCHEME}
+
+	# WebSocket proxy
+	ProxyPass "/api/v1/streaming" "ws://127.0.0.1:4000"
+
+	ErrorDocument 500 /500.html
+	ErrorDocument 501 /500.html
+	ErrorDocument 502 /500.html
+	ErrorDocument 503 /500.html
+	ErrorDocument 504 /500.html
+
+  <LocationMatch "^/(emoji|packs|system/accounts/avatars|system/media_attachments/files)">
+    ExpiresActive on
+    ExpiresDefault "access plus 1 year"
+  </LocationMatch>
+
+  <Location "/sw.js">
+    ExpiresActive on
+    ExpiresDefault "access"
+  </Location>
+
+	#   SSL Engine Switch:
+	#   Enable/Disable SSL for this virtual host.
+	SSLEngine on
+
+  # Specify here the paths to letsencrypt's certificate.
+	#   If both key and certificate are stored in the same file, only the
+	#   SSLCertificateFile directive is needed.
+	SSLCertificateFile    /etc/ssl/private/fullchain.pem
+	SSLCertificateKeyFile /etc/ssl/private/key.pem
+
+	#   SSL Protocol Adjustments:
+	#   The safe and default but still SSL/TLS standard compliant shutdown
+	#   approach is that mod_ssl sends the close notify alert but doesn't wait for
+	#   the close notify alert from client. When you need a different shutdown
+	#   approach you can use one of the following variables:
+	#   o ssl-unclean-shutdown:
+	#     This forces an unclean shutdown when the connection is closed, i.e. no
+	#     SSL close notify alert is send or allowed to received.  This violates
+	#     the SSL/TLS standard but is needed for some brain-dead browsers. Use
+	#     this when you receive I/O errors because of the standard approach where
+	#     mod_ssl sends the close notify alert.
+	#   o ssl-accurate-shutdown:
+	#     This forces an accurate shutdown when the connection is closed, i.e. a
+	#     SSL close notify alert is send and mod_ssl waits for the close notify
+	#     alert of the client. This is 100% SSL/TLS standard compliant, but in
+	#     practice often causes hanging connections with brain-dead browsers. Use
+	#     this only for browsers where you know that their SSL implementation
+	#     works correctly.
+	#   Notice: Most problems of broken clients are also related to the HTTP
+	#   keep-alive facility, so you usually additionally want to disable
+	#   keep-alive for those clients, too. Use variable "nokeepalive" for this.
+	#   Similarly, one has to force some clients to use HTTP/1.0 to workaround
+	#   their broken HTTP/1.1 implementation. Use variables "downgrade-1.0" and
+	#   "force-response-1.0" for this.
+	BrowserMatch "MSIE [2-6]" \
```

Comment:
Mastodon only supports IE 11 or newer. https://github.com/tootsuite/mastodon/blob/master/package.json#L25

Reply:
I removed this line, thanks for the notice.